projects/ngx-charts-on-fhir/src/lib/fhir-chart/fhir-chart.component.ts

```diff
@@ -26,6 +26,8 @@ export class FhirChartComponent implements OnInit {
   @Input() width: string = '600px';
   @Input() height: string = '300px';
 
+  @ViewChild('baseChart') baseChart?:ElementRef;
```

Comment:
Is the @ViewChild still needed?

Reply:
No,@ViewChild is not needed Now. I will remove that.